torchgen/static_runtime/generator.py

```diff
@@ -203,6 +203,11 @@ def has_alias(
         "alias_copy",
         "_triton_multi_head_attention",
         "special_airy_ai",
+        "special_betainc",
```

Comment:
I don't think static_runtime is used anymore. Let's keep these functions out of this to not change existing systems.